### PR TITLE
Add nt660 PCB

### PR DESCRIPTION
**Pull request by Maartenwut**
## Description
Adds the nt660 PCB to QMK. Has my keymap as well as a default and a VIA enabled keymap. This PCB is a direct replacement for the Leopold FC660m.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by Maartenwut** [comment]
It's also known as the mt660, but due to there already being a mt980 in qmk I decided to rename it.

File: keyboards/maartenwut/nt660/config.h [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4705
+#define PRODUCT_ID      0x1f02
```
Comment (fauxpark) [inline keyboards/maartenwut/nt660/config.h:24]:
```suggestion
#define PRODUCT_ID      0x1F02
```

File: keyboards/maartenwut/nt660/keymaps/maartenwut/keymap.c [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  _______, _______, _______, _______, _______, TG(_GA), _______, _______, _______, _______, _______, _______,  _______, _______,             \
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_VOLD, KC_VOLU, KC_MUTE,  _______, _______,             \
+  _______, _______, _______,                   _______,                            _______, _______, _______,  _______, _______, _______),
+  
+[_SP] = LAYOUT_all(
+  KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,   KC_F11,  KC_F12,  _______, _______,    \
+  _______, _______, _______, _______, _______, _______, _______, _______, KC_UP,   _______, _______, _______,  _______, _______, _______,    \
+  _______, _______, _______, _______, _______, _______, _______, KC_LEFT, KC_DOWN, KC_RGHT, _______, _______,  _______, _______,             \
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,  _______, _______,             \
+  _______, _______, _______,                   _______,                            _______, _______, _______,  _______, _______, _______),
```
Comment (fauxpark) [inline keyboards/maartenwut/nt660/keymaps/maartenwut/keymap.c:57]:
```suggestion
[_MA] = LAYOUT_all(
  KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS,  KC_EQL,  KC_BSPC, KC_MPLY,
  KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC,  KC_RBRC, KC_BSLS, KC_DEL,
  KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,  _______, KC_ENT,
  KC_LSFT, _______, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,  KC_RSFT, KC_UP,
  KC_LCTL, KC_LGUI, KC_LALT,                   SPACE,                              KC_RALT, KC_RCTRL,MO(_FL),  KC_LEFT, KC_DOWN, KC_RGHT),
  
[_GA] = LAYOUT_all(
  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,  _______, _______, _______,
  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,  _______, _______, _______,
  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,  _______, _______,
  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,  _______, _______,
  _______, _______, _______,                   KC_SPC,                             _______, _______, _______,  _______, _______, _______),
  
[_FL] = LAYOUT_all(
  KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,   KC_F11,  KC_F12,  RESET,   KC_PSCR,
  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,  _______, _______, KC_INS,
  _______, _______, _______, _______, _______, TG(_GA), _______, _______, _______, _______, _______, _______,  _______, _______,
  _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_VOLD, KC_VOLU, KC_MUTE,  _______, _______,
  _______, _______, _______,                   _______,                            _______, _______, _______,  _______, _______, _______),
  
[_SP] = LAYOUT_all(
  KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,   KC_F11,  KC_F12,  _______, _______,
  _______, _______, _______, _______, _______, _______, _______, _______, KC_UP,   _______, _______, _______,  _______, _______, _______,
  _______, _______, _______, _______, _______, _______, _______, KC_LEFT, KC_DOWN, KC_RGHT, _______, _______,  _______, _______,
  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,  _______, _______,
  _______, _______, _______,                   _______,                            _______, _______, _______,  _______, _______, _______),
```

File: keyboards/maartenwut/nt660/info.json [hunk 3]
```diff
@@ -0,0 +1,158 @@
+{
+    "keyboard_name": "nt660", 
+    "url": "", 
+    "maintainer": "maartenwut", 
+    "width": 16.5, 
+    "height": 5, 
+    "layouts": {
+        "LAYOUT_66_ansi": {
```
Comment (noroadsleft) [inline keyboards/maartenwut/nt660/info.json:8]:
```suggestion
        "LAYOUT_all": {
            "layout": [
                {"label":"k00", "x":0, "y":0},
                {"label":"k01", "x":1, "y":0},
                {"label":"k02", "x":2, "y":0},
                {"label":"k03", "x":3, "y":0},
                {"label":"k04", "x":4, "y":0},
                {"label":"k05", "x":5, "y":0},
                {"label":"k06", "x":6, "y":0},
                {"label":"k07", "x":7, "y":0},
                {"label":"k08", "x":8, "y":0},
                {"label":"k09", "x":9, "y":0},
                {"label":"k0a", "x":10, "y":0},
                {"label":"k0b", "x":11, "y":0},
                {"label":"k0c", "x":12, "y":0},
                {"label":"k0f", "x":13, "y":0, "w":2},
                {"label":"k0d", "x":15.5, "y":0},

                {"label":"k10", "x":0, "y":1, "w":1.5},
                {"label":"k11", "x":1.5, "y":1},
                {"label":"k12", "x":2.5, "y":1},
                {"label":"k13", "x":3.5, "y":1},
                {"label":"k14", "x":4.5, "y":1},
                {"label":"k15", "x":5.5, "y":1},
                {"label":"k16", "x":6.5, "y":1},
                {"label":"k17", "x":7.5, "y":1},
                {"label":"k18", "x":8.5, "y":1},
                {"label":"k19", "x":9.5, "y":1},
                {"label":"k1a", "x":10.5, "y":1},
                {"label":"k1b", "x":11.5, "y":1},
                {"label":"k1c", "x":12.5, "y":1},
                {"label":"k1f", "x":13.5, "y":1, "w":1.5},
                {"label":"k1d", "x":15.5, "y":1},

                {"label":"k20", "x":0, "y":2, "w":1.75},
                {"label":"k21", "x":1.75, "y":2},
                {"label":"k22", "x":2.75, "y":2},
                {"label":"k23", "x":3.75, "y":2},
                {"label":"k24", "x":4.75, "y":2},
                {"label":"k25", "x":5.75, "y":2},
                {"label":"k26", "x":6.75, "y":2},
                {"label":"k27", "x":7.75, "y":2},
                {"label":"k28", "x":8.75, "y":2},
                {"label":"k29", "x":9.75, "y":2},
                {"label":"k2a", "x":10.75, "y":2},
                {"label":"k2b", "x":11.75, "y":2},
                {"label":"k2c", "x":12.75, "y":2},
                {"label":"k2f", "x":13.75, "y":2, "w":1.25},

                {"label":"k30", "x":0, "y":3, "w":1.25},
                {"label":"k31", "x":1.25, "y":3},
                {"label":"k32", "x":2.25, "y":3},
                {"label":"k33", "x":3.25, "y":3},
                {"label":"k34", "x":4.25, "y":3},
                {"label":"k35", "x":5.25, "y":3},
                {"label":"k36", "x":6.25, "y":3},
                {"label":"k37", "x":7.25, "y":3},
                {"label":"k38", "x":8.25, "y":3},
                {"label":"k39", "x":9.25, "y":3},
                {"label":"k3a", "x":10.25, "y":3},
                {"label":"k3b", "x":11.25, "y":3},
                {"label":"k3c", "x":12.25, "y":3, "w":2.25},
                {"label":"k3f", "x":14.5, "y":3},

                {"label":"k40", "x":0, "y":4, "w":1.25},
                {"label":"k41", "x":1.25, "y":4},
                {"label":"k42", "x":2.25, "y":4, "w":1.25},
                {"label":"k46", "x":3.5, "y":4, "w":6.25},
                {"label":"k49", "x":9.75, "y":4, "w":1.25},
                {"label":"k4a", "x":11, "y":4, "w":1.25},
                {"label":"k4b", "x":12.25, "y":4, "w":1.25},
                {"label":"k4c", "x":13.5, "y":4},
                {"label":"k4f", "x":14.5, "y":4},
                {"label":"k4d", "x":15.5, "y":4}
            ]
        },
        "LAYOUT_66_ansi": {
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**